Makefile.toml

```diff
@@ -60,7 +60,7 @@ workspace = false
 [tasks.run-local-integration-tests-debug]
 description = "fastest way to build and debug runtime in simulator"
 script = '''
-RUST_BACKTRACE=full RUST_LOG=trace,parity-db=warn,trie=warn,runtime=trace,substrate-relay=trace,bridge=trace,xcmp=trace cargo +stable test  --package local-integration-tests --features local-integration-tests --features picasso  --no-default-features -- --nocapture --test-threads=1
+RUST_BACKTRACE=full RUST_LOG=trace,parity-db=warn,trie=warn,runtime=trace,substrate-relay=trace,bridge=trace,xcmp=trace,xcm=trace cargo +stable test  --package local-integration-tests --features local-integration-tests --features picasso  --no-default-features -- --nocapture --test-threads=1
```

Comment:
This is quite a long command, perhaps split this using the `env` key for the task?